Description: Use Debian CFLAGS and LDFLAGS as well as math library
Author: Jacob Adams

--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
-CFLAGS := $(shell sdl-config --cflags) -Wall -g
-LDLIBS := $(shell sdl-config --libs) -lSDLmain -lSDL_image -lSDL_ttf -lSDL_mixer
+CFLAGS := $(shell dpkg-buildflags --get CFLAGS) $(shell sdl-config --cflags) -Wall -g
+LDLIBS := $(shell dpkg-buildflags --get LDFLAGS) $(shell sdl-config --libs) -lSDLmain -lSDL_image -lSDL_ttf -lSDL_mixer -lm
 
 default: beret
 
